tools/converters/excel/Converters/StringConverters.cs

```diff
@@ -30,26 +31,37 @@ public static class StringConverters
         }
 
         /// <summary>
-        /// 
+        ///
         /// </summary>
         /// <param name="value"></param>
         /// <param name="nullable"></param>
         /// <returns></returns>
         public static DateTime? ConvertToDate(this string value, bool nullable = false)
         {
-            if (Regex.Match(value, "[0-9]{2}/[0-9]{2}").Success)
+            if (Regex.Match(value, "^[0-9]{2,4}/[0-9]{2,4}$").Success)
```

Comment:
I wonder if this could be converted to use tryparse instead:
https://docs.microsoft.com/en-us/dotnet/api/system.datetime.tryparse?view=net-5.0
not important for now, maybe tech debt.